src/Java.Interop.Tools.JavaCallableWrappers/Java.Interop.Tools.JavaCallableWrappers.CallableWrapperMembers/CallableWrapperConstructor.cs

```diff
@@ -0,0 +1,75 @@
+using System.IO;
+
+namespace Java.Interop.Tools.JavaCallableWrappers.CallableWrapperMembers;
+
+public class CallableWrapperConstructor : CallableWrapperMethod
+{
+	public bool CannotRegisterInStaticConstructor { get; set; }
```

Comment:
Instead of these two fields, I think I'd prefer to have a `public CallableWrapperType? DeclaringType {get;set;}` property, then read `CannotRegisterInStaticConstructor` from e.g. `DeclaringType.CannotRegisterInStaticConstructor`.  This would help ensure consistency, as these values *cannot* differ from the declaring type.

Reply:
Fixed